upgrade/constants.go

```diff
@@ -34,5 +34,62 @@ const UpgradeASCII = `
 // TestnetChainID is the Chain ID of the Noble testnet.
 const TestnetChainID = "grand-1"
 
+// TestnetHyperlaneDomain is the Hyperlane domain of the Noble testnet.
+// Generated from: console.log(parseInt('0x'+Buffer.from('GRAN').toString('hex')))
+// We truncate "GRAND" to "GRAN" to not exceed the uint32 maximum.
+const TestnetHyperlaneDomain = 1196573006
+
 // MainnetChainID is the Chain ID of the Noble mainnet.
 const MainnetChainID = "noble-1"
+
+// MainnetHyperlaneDomain is the Hyperlane domain of the Noble mainnet.
+// Generated from: console.log(parseInt('0x'+Buffer.from('NOBL').toString('hex')))
+// We truncate "NOBLE" to "NOBL" to not exceed the uint32 maximum.
+const MainnetHyperlaneDomain = 1313817164
+
+// HyperlaneDefaultISMs defines the default Hyperlane ISMs for external chains
+// that Noble will support at launch via a routing ISM.
+var HyperlaneDefaultISMs = map[string][]struct {
+	Domain     uint32
+	Name       string
+	Validators []string
+	Threshold  uint32
+}{
+	TestnetChainID: {
+		// TODO: auroratestnet
```

Comment:
For reviewers, just waiting on the Hyperlane team to confirm their ISM validator on testnet!